app/controllers/settings/password.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  actions: {
+    changePassword(passwordData) {
+      this.set('isLoading', true);
+      let payload = {
+        'data': {
+          'old-password' : passwordData.passwordCurrent,
+          'new-password' : passwordData.passwordNew
+        }
+      };
+      this.get('loader')
+        .post('/auth/change-password', payload)
+        .then(data => {
+          if (data.error) {
```

Comment:
Please make modifications to the loader service to pass down the json response or plain response if it exists.